### PR TITLE
Added test and benchmark for consistent read

### PR DESCRIPTION
**Pull request by ateleshev**
**What this PR does / why we need it**:
This PR gives testing ability for consistent read

### CODE REVIEW

File: pkg/util/io/consistentread_test.go [hunk 1]
```diff
@@ -0,0 +1,411 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```
Comment (neolit123) [inline pkg/util/io/consistentread_test.go]: 2018

File: pkg/util/io/consistentread_test.go [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+package io
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var content = []byte( // {{{
```
Comment (neolit123) [inline pkg/util/io/consistentread_test.go]: `var` -> `const`

Comment (neolit123) [reply pkg/util/io/consistentread_test.go]: does the content / input have to be that detailed? :)

File: pkg/util/io/consistentread_test.go [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+shm /var/lib/docker/containers/9b39e9d943c51ea380c5f842aefd665b788087357db16f3f3f2c2fe35f7a8cbb/mounts/shm tmpfs rw,seclabel,nosuid,nodev,noexec,relatime,size=65536k 0 0
+tmpfs /var/lib/kubelet/pods/99ed52ad-8997-11e8-97bb-002590e724c4/volumes/kubernetes.io~secret/default-token-lhldf tmpfs rw,seclabel,relatime 0 0
+overlay /var/lib/docker/overlay2/d91645ac1b84930d2ea114b87effe553db50315ac11d5073876b59c10ddcfb19/merged overlay rw,relatime,lowerdir=/var/lib/docker/overlay2/l/J5GYAHHW5MDGCFJQBLESILZYUO:/var/lib/docker/overlay2/l/EVC74UM4Y4HWBLDPV7AAO2MBXH:/var/lib/docker/overlay2/l/GDJBND6DYLMC55TLYARSFPRGUX:/var/lib/docker/overlay2/l/AZDKNLHIYZYRQASJCIIU6IECG6,upperdir=/var/lib/docker/overl
```
Comment (neolit123) [inline pkg/util/io/consistentread_test.go]: some empty lines can be omitted if the previous line has a `{`.

**Comment by neolit123** [comment]
thanks for this PR @ateleshev !
please make sure that your run `./hack/update-bazel.sh` and include the updated BUILD file.
also you can run things like:
```
./hack/verify-gofmt.sh
./hack/verify-golint.sh
./hack/verify-govet.sh
```
to do some local verification tests.

**Comment by ateleshev** [comment]
@neolit123 done, please review again

**Comment by neolit123** [comment]
@ateleshev thanks
/area test
/ok-to-test

**Pull request closed without merging**